httpd/handler_test.go

```diff
@@ -1615,10 +1615,73 @@ func TestSnapshotHandler(t *testing.T) {
 	}
 }
 
+// Ensure that the server will stream out results if a chunked response is requested
+func TestHandler_ChunkedResponses(t *testing.T) {
+	c := test.NewMessagingClient()
+	defer c.Close()
+	srvr := OpenAuthlessServer(c)
+	srvr.CreateDatabase("foo")
+	srvr.CreateRetentionPolicy("foo", influxdb.NewRetentionPolicy("bar"))
+	srvr.SetDefaultRetentionPolicy("foo", "bar")
+
+	s := NewHTTPServer(srvr)
+	defer s.Close()
+
+	status, errString := MustHTTP("POST", s.URL+`/write`, nil, nil, `{"database" : "foo", "retentionPolicy" : "bar", "points": [
+			{"name": "cpu", "tags": {"host": "server01"},"timestamp": "2009-11-10T23:00:00Z", "fields": {"value": 100}},
+			{"name": "cpu", "tags": {"host": "server02"},"timestamp": "2009-11-10T23:30:00Z", "fields": {"value": 25}}]}`)
+	if status != http.StatusOK {
+		t.Fatalf("unexpected status: %d - %s", status, errString)
+	}
+	time.Sleep(100 * time.Millisecond) // Ensure data node picks up write.
```

Comment:
that seems tricky because of how chunking works. I would really have to do the write and then wait for the regular raw query to return with the data. Then run the chunked query test.